### PR TITLE
https://issues.redhat.com/browse/ACM-11476 Clarify Nutanix support CIM

### PR DESCRIPTION
**Pull request by oafischer**
Also for 2.10.

### CODE REVIEW

File: clusters/cluster_lifecycle/cim_intro.adoc [hunk 1]
```diff
@@ -18,6 +18,8 @@ After the host boots and the agent contacts the service, the service creates a n
 
 You can install hosts in the inventory as OpenShift nodes later. The agent writes the operating system to the disk, along with the necessary configuration, and reboots the host.
 
+*Note:* {product-title-short} 2.9 and later and central infrastructure management support Nutanix by using `AgentClusterInstall`, which might have some limitations. To learn more, see link:https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2024/html/installing_openshift_container_platform_with_the_assisted_installer/assembly_installing-on-nutanix[Optional: Installing on Nutanix] in the Assisted Installer documentation.
```
Comment (racedo) [inline clusters/cluster_lifecycle/cim_intro.adoc]: Replace 'might have some limitations' with the requirements. Essentially, users must use ACM's CIM and users must create the Nutanix Virtual Machines before installing OpenShift on Nutanix.

Comment (oafischer) [reply clusters/cluster_lifecycle/cim_intro.adoc]: Changed it to "additional requirements". Let me know if that works. Thanks!

File: clusters/cluster_lifecycle/cim_intro.adoc [hunk 2]
```diff
@@ -18,6 +18,8 @@ After the host boots and the agent contacts the service, the service creates a n
 
 You can install hosts in the inventory as OpenShift nodes later. The agent writes the operating system to the disk, along with the necessary configuration, and reboots the host.
 
+*Note:* {product-title-short} 2.9 and later and central infrastructure management support Nutanix by using `AgentClusterInstall`, which requires additional configuration. To learn more, see link:https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2024/html/installing_openshift_container_platform_with_the_assisted_installer/assembly_installing-on-nutanix[Optional: Installing on Nutanix] in the Assisted Installer documentation.
```
Comment (cadawson) [inline clusters/cluster_lifecycle/cim_intro.adoc]:
```suggestion
*Note:* {product-title-short} 2.9 and later and central infrastructure management support the Nutanix platform by using `AgentClusterInstall`, which requires additional configuration by creating the Nutanix virtual machines. To learn more, see link:https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2024/html/installing_openshift_container_platform_with_the_assisted_installer/assembly_installing-on-nutanix[Optional: Installing on Nutanix] in the Assisted Installer documentation.
```

**Review by cadawson — COMMENTED** [review]
Looks good. Added a couple of suggestions. Thanks!

**Pull request merged**